t/test-lib.sh

```diff
@@ -37,16 +37,13 @@ then
 fi
 STG_ROOT=$(cd "$TEST_DIRECTORY"/.. && pwd) || exit 1
 export STG_ROOT
-stg_build_dir="$STG_ROOT"/build
 PYTHON="${PYTHON:-python}"
-python_major_minor="$($PYTHON -c '
-import sys
-print(".".join(map(str, sys.version_info[:2])))')"
-stg_bin_dir="$stg_build_dir"/scripts-"$python_major_minor"
+stg_build_dir="$STG_ROOT/build/lib"
+stg_t_dir="$STG_ROOT/t"
 
-if test ! -f "$stg_bin_dir"/stg
+if test ! -f "$stg_build_dir/stgit/main.py"
 then
-	echo >&2 "error: $stg_bin_dir missing (has stg been built?)."
+	echo >&2 "error: $stg_build_dir/stgit/main.py missing (has stg been built?)."
```

Comment:
same as above -- we should avoid changes to `test-lib.sh` since it's regularly merged from `git`

Reply:
Hmm, without having checked, I somehow doubt that the git test infra contains `stg_bin_dir` as a variable.

Reply:
That's a good point, this is stuff we add. Ignore this comment